cpp/src/ray/util/process_helper.h

```diff
@@ -1,11 +1,24 @@
 #pragma once
+#include <ray/api/ray_config.h>
+#include <string>
+#include "ray/core.h"
 
 namespace ray {
 namespace api {
 
 class ProcessHelper {
  public:
-  static void RayStart();
+  void RayStart(std::shared_ptr<RayConfig> config);
+  void RayStop(std::shared_ptr<RayConfig> config);
+  static std::shared_ptr<ProcessHelper> getInstance() {
+    if (ProcessHelper_ == nullptr) {
```

Comment:
Is it a singleton?
If it's a singleton, use the c++11 style singleton:https://stackoverflow.com/questions/1008019/c-singleton-design-pattern